### DISCUSSION TITLE
Think about adding a progress bar

### DISCUSSION THREAD

**Discussion by szaimen** [nextcloud/all-in-one#1099] [Ideas] [answered]
Maybe just put a timer of 5min there. If it is done earlier it is fine

**Comment by szaimen** [comment, marked as answer]
Another option would be showing which container currently gets downloaded/started but not sure how to give that info back to the frontend...

**Comment by d-513** [comment]
This would be useful. Definietly, don't add a timer, that's just confusing.

Instead, I think the best option would be to show a log with what container is being updated and what is being done, the log from the container, etc, all that important stuff. Send it to frontend via websocket.

I don't know if it's stuck or what when it's updating, it would be a better user experience to tell them what it is doing
